<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modbus_slave</class>
 <widget class="QWidget" name="modbus_slave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>modbus_slave</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_coil">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>线圈状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>保持寄存器：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_holdregiter">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonStart">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>470</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>启动</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
